store/storetest/cluster_discovery_store.go

```diff
@@ -61,39 +57,33 @@ func testClusterDiscoveryStoreLastPing(t *testing.T, ss store.Store) {
 		Type:        "test_test_lastPing" + model.NewId(),
 	}
 
-	if err := ss.ClusterDiscovery().Save(discovery); err != nil {
-		t.Fatal(err)
-	}
+	err := ss.ClusterDiscovery().Save(discovery)
+	require.Nil(t, err)
 
-	if err := ss.ClusterDiscovery().SetLastPingAt(discovery); err != nil {
-		t.Fatal(err)
-	}
+	err = ss.ClusterDiscovery().SetLastPingAt(discovery)
+	require.Nil(t, err)
 
 	ttime := model.GetMillis()
 
 	time.Sleep(1 * time.Second)
 
-	if err := ss.ClusterDiscovery().SetLastPingAt(discovery); err != nil {
-		t.Fatal(err)
-	}
+	err = ss.ClusterDiscovery().SetLastPingAt(discovery)
+	require.Nil(t, err)
 
 	list, err := ss.ClusterDiscovery().GetAll(discovery.Type, "cluster_name_lastPing")
 	require.Nil(t, err)
 	assert.Len(t, list, 1)
 
-	if list[0].LastPingAt-ttime < 500 {
-		t.Fatal("failed to set time")
-	}
+	require.Less(t, list[0].LastPingAt-ttime, int64(500))
```

Comment:
You mixed up the ordering. See https://godoc.org/github.com/stretchr/testify/assert#Less

Reply:
ohhh nice catch thanks @hanzei

Reply:
Your welcome